kili/services/label_import/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+label import service
+"""
+
+from pathlib import Path
+from typing import List, Optional, Type, cast
+
+from kili.exceptions import NotFound
+from kili.services.label_import.importer import (
+    AbstractLabelImporter,
+    KiliRawLabelImporter,
+    LoggerParams,
+    YoloLabelImporter,
+)
+from kili.services.types import LabelFormat, LogLevel
+
+
+def import_labels_from_files(  # pylint: disable=too-many-arguments
+    kili,
+    labels_file_path: Optional[str],
+    labels_files: Optional[List[str]],
+    meta_file_path: Optional[str],
+    project_id: str,
+    input_format: str,
+    target_job_name: Optional[str],
+    disable_tqdm: bool,
+    log_level: str,
+    model_name: Optional[str],
+    is_prediction: bool,
+) -> None:
+    """
+    Imports labels from a list of files contained in file path.
+    """
+    if kili.count_projects(project_id=project_id) == 0:
+        raise NotFound(f"project ID: {project_id}")
+
+    if is_prediction and model_name is None:
+        raise ValueError("If predictions are uploaded, a model name should be specified")
+
+    label_importer_class: Optional[Type[AbstractLabelImporter]] = None
+    if input_format in ["yolo_v4", "yolo_v5"]:
+        label_importer_class = YoloLabelImporter
+    elif input_format == "raw":
+        label_importer_class = KiliRawLabelImporter
+    else:
+        raise NotImplementedError(f"{input_format} import is not implemented yet.")
+
+    logger_params = LoggerParams(disable_tqdm=disable_tqdm, level=cast(LogLevel, log_level))
+    label_importer = label_importer_class(kili, logger_params, cast(LabelFormat, input_format))
+    label_importer.process_from_files(
+        Path(labels_file_path) if labels_file_path is not None else None,
```

Comment:
labels_file_path and Path(labels_file_path)

(same below)

Reply:
it does not work with typing (labels_path_path can be a string)